content/en/blog/2023/jmx-metric-insight/index.md

```diff
@@ -0,0 +1,225 @@
+---
+title: JMX Metric Insight
+linkTitle: JMX Metric Insight
+date: 2023-01-11
+author: "[Samudraneel Dasgupta](https://github.com/Samudraneel24) (Cisco)"
+---
+
+[JMX](https://www.oracle.com/technical-resources/articles/javase/jmx.html) (Java
+Management Extensions) is a technology that provides a way to manage and monitor
+Java-based applications. Detailed information about the performance and resource
+usage of the application can be derived from JMX metrics. This data can help you
+identify trends or potential issues with the application and take action to
+address them before they become serious problems. If there are any problems, we
+can diagnose them with the help of metrics collected, and fine-tune the system
+for optimal performance.
+
+With the addition of the
+[JMX Metric Insight](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/jmx-metrics)
+module into the opentelemetry-java-instrumentation agent, we don't need to
+deploy a separate service just to collect JMX metrics for monitoring our
+application. The agent can now natively collect and export metrics exposed by
+application servers through local
+[MBeans](https://docs.oracle.com/javase/tutorial/jmx/mbeans/index.html)
+available within the instrumented application. The required MBeans and
+corresponding metrics can be described using a YAML configuration file. The
+individual metric configurations allow precise metric selection and
+identification. JMX Metric Insight comes with a number of predefined
+configurations containing curated sets of JMX metrics for popular application
+servers or frameworks, such as
+[ActiveMQ](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/jmx-metrics/javaagent/activemq.md),
+[Hadoop](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/jmx-metrics/javaagent/hadoop.md),
+[Jetty](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/jmx-metrics/javaagent/jetty.md),
+[Kafka Broker](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/jmx-metrics/javaagent/kafka-broker.md),
+[Tomcat](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/jmx-metrics/javaagent/tomcat.md)
+and
+[Wildfly](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/jmx-metrics/javaagent/wildfly.md).
+You can also provide your own metric definitions, through one or more YAML
+files.
```

Comment:
Add a pointer to the docs where people can learn how to write their own YAML file